packages/mobile/android/app/build.gradle

```diff
@@ -182,65 +186,65 @@ android {
     splits {
         abi {
             reset()
-            enable false
-            universalApk false  // If true, also generate a universal APK
+            enable = false
+            universalApk = false  // If true, also generate a universal APK
             //noinspection ChromeOsAbiSupport
-            include (*reactNativeArchitectures())
+            include(*reactNativeArchitectures())
         }
     }
     signingConfigs {
         debug {
-            storeFile file('debug.keystore')
-            storePassword 'android'
-            keyAlias 'androiddebugkey'
-            keyPassword 'android'
+            storeFile = file("debug.keystore")
+            storePassword = "android"
+            keyAlias = "androiddebugkey"
+            keyPassword = "android"
```

Comment:
these diffs are annoying, but switching from implicit assignments to explicit with equals sign,  using explicit parentheses on groovy method invocations in the gradle, and using double quotes in place of single ones is the first step from migrating the build system away from groovy and to the newly supported (and way less annoying to use) Kotlin DSL.